test/emptytheme/functions.php

```diff
@@ -30,3 +30,32 @@ function emptytheme_scripts() {
 	}
 	add_action( 'wp_enqueue_scripts', 'emptytheme_scripts' );
 endif;
+
+if ( ! function_exists( 'emptytheme_register_custom_fields' ) ) :
```

Comment:
That probably should be a custom plugin that you activate only when running the tests.

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/58550/commits/50ad1966644ae76d10851389c05d8928799fb2f7).